Sourcery/Parsing/Utils/AnnotationsParser.swift

```diff
@@ -92,6 +92,15 @@ internal struct AnnotationsParser {
         // `case` is not included in the key of enum case definition, so we strip it manually
         prefix = prefix.trimmingSuffix("case").trimmingCharacters(in: .whitespaces)
 
+        // Strips the access level if any
+        if let accessLevel = AccessLevel.all
```

Comment:
I guess the issue happens not only when using access level, but i.e. using `final` or any attributes on the same line with declaration. We can of course add it to the list of prefixes to trim, but it does not look like bulletproof solution...